public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -49,9 +49,43 @@ data class EntitlementInfo constructor(
     @Deprecated(
         "Use rawData instead",
         replaceWith = ReplaceWith("rawData")
-    ) val jsonObject: JSONObject
+    ) val jsonObject: JSONObject,
+    val verification: VerificationResult
 ) : Parcelable, RawDataContainer<JSONObject> {
 
+    @Deprecated("Use full constructor instead")
```

Comment:
As we discussed over Slack, I'm adding a deprecated constructor with a default value for now. Reason to not simply add a default value to the field is that I don't think we should allow creating these models without this object, and adding a default value can lead to forgetting adding that field. This way at least we would get a deprecated warning.